packages/core/src/fetch.ts

```diff
@@ -205,6 +205,7 @@ function endSpan(span: Span, handlerData: HandlerDataFetch): void {
   if (handlerData.response) {
     setHttpStatus(span, handlerData.response.status);
 
+    // fixme: only send if `sendDefaultPii`?
```

Comment:
I think content length is something we could send

Reply:
you mean w/o sendDefaultPii === true? agreed!